code/game/objects/items/storage/pouch.dm

```diff
@@ -374,11 +375,11 @@
 
 /obj/item/storage/pouch/flare/full/New()
 	..()
-	new /obj/item/device/flashlight/flare(src)
-	new /obj/item/device/flashlight/flare(src)
-	new /obj/item/device/flashlight/flare(src)
-	new /obj/item/device/flashlight/flare(src)
-	new /obj/item/device/flashlight/flare(src)
+	new /obj/item/explosive/grenade/flare(src)
```

Comment:
Make an explosive nades pouch if you want, but don't remove the old flares.

Reply:
Old flares remain in the game, but the military standard should be the flare grenade.

Reply:
A flare is not an effective weapon.

Reply:
It sets people on fire, no?